tests/testproject/testproject/settings.py

```diff
@@ -78,11 +78,11 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'relativedelta-testproject',
-        'HOST': os.environ.get('PG_HOST', '127.0.0.1'),
-        'PORT': os.environ.get('PG_PORT', 5432),
-        'USER': os.environ.get('PG_USER', 'postgres'),
-        'PASSWORD': os.environ.get('PG_PASSWORD', '')
+        'NAME': os.environ.get('PGDATABASE', 'relativedelta-testproject'),
+        'HOST': os.environ.get('PGHOSTADDR', os.environ.get('PGHOST', '127.0.0.1')),
```

Comment:
If you change it to do `os.environ.get('PGHOSTADDR')` it will use `None` when it's not supplied, which then falls back to the `libpq` defaults (which is the UNIX domain socket). If done for `PGUSER` as well, it'll use the current UNIX user if `None`.

Reply:
Should it read:
```'HOST': os.environ.get('PGHOSTADDR', os.environ.get('PGHOST', None')),``` 

Can you test it on your end as I'm not using a local postgres?

Reply:
This works for me:

```
diff --git a/tests/testproject/testproject/settings.py b/tests/testproject/testproject/settings.py
index 342426a..5bd0b9e 100644
--- a/tests/testproject/testproject/settings.py
+++ b/tests/testproject/testproject/settings.py
@@ -79,10 +79,10 @@ DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': os.environ.get('PGDATABASE', 'relativedelta-testproject'),
-        'HOST': os.environ.get('PGHOSTADDR', os.environ.get('PGHOST', '127.0.0.1')),
-        'PORT': os.environ.get('PGPORT', 5432),
-        'USER': os.environ.get('PGUSER', 'postgres'),
-        'PASSWORD': os.environ.get('PGPASSWORD', '')
+        'HOST': os.environ.get('PGHOSTADDR', os.environ.get('PGHOST')),
+        'PORT': os.environ.get('PGPORT'),
+        'USER': os.environ.get('PGUSER'),
+        'PASSWORD': os.environ.get('PGPASSWORD', os.environ.get('PGPASS'))
     },
 }
```

Reply:
If you can change this one last thing I'll merge the PR

Reply:
Done now

Reply:
Awesome, thanks for your patience!